pkgs/applications/blockchains/polkadot/substrate-wasm-builder-runner.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/Cargo.lock b/Cargo.lock
+index 5e7c4a14..bb67aada 100644
+--- a/Cargo.lock
++++ b/Cargo.lock
+@@ -8642,8 +8642,7 @@ dependencies = [
+ [[package]]
+ name = "substrate-wasm-builder-runner"
+ version = "1.0.6"
+-source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "d2a965994514ab35d3893e9260245f2947fd1981cdd4fffd2c6e6d1a9ce02e6a"
++source = "git+https://github.com/paritytech/substrate#647ad15565d7c35ecf00b73b12cccad9858780b9"
+ 
+ [[package]]
+ name = "subtle"
+diff --git a/Cargo.toml b/Cargo.toml
+index 78047a1a..2d571f8e 100644
+--- a/Cargo.toml
++++ b/Cargo.toml
+@@ -112,3 +112,6 @@ polkadot = { path = "/usr/bin/polkadot" }
+ 
+ [package.metadata.rpm.files]
+ "../scripts/packaging/polkadot.service" = { path = "/usr/lib/systemd/system/polkadot.service", mode = "644" }
++
++[patch.crates-io]
++substrate-wasm-builder-runner = { git = "https://github.com/paritytech/substrate", branch = "master" }
```

Comment:
just noticed that this follows master, but that's not a stable endpoint... an update to their master will break the build. Please use a commit instead

Reply:
https://github.com/NixOS/nixpkgs/pull/98785/files#diff-71369739716eac7dd99e9da07e1f4944R11

That's done here in the `Cargo.lock`.